superbench/executor/executor.py

```diff
@@ -174,6 +198,18 @@ def exec(self):
             benchmark_config = self._sb_benchmarks[benchmark_name]
             benchmark_results = list()
             self.__create_benchmark_dir(benchmark_name)
+
+            monitor = None
+            if self.__get_rank_id() == 0 and self._sb_monitor_config and self._sb_monitor_config.enable:
+                if self.__get_platform() == Platform.CUDA:
+                    monitor = Monitor(
+                        None, int(self._sb_monitor_config.sample_duration or 10),
+                        int(self._sb_monitor_config.sample_freq or 1), self.__get_monitor_path(benchmark_name)
+                    )
+                    monitor.start()
+                else:
+                    logger.warning('Monitor can not support ROCM platform.')
```

Comment:
else includes Platform.CPU? do you want to monitor cpu and network because only gpu metrics won't work?

Reply:
Updated the log message.